Cred store docs days (#4579)
hashicorp/boundary: website/content/docs/configuration/credential-management/configure-credential-brokering.mdx

---
layout: docs
page_title: Configure targets with credential brokering
description: |-
  How to configure targets with credential brokering
---

# Configure targets with credential brokering

> The following section provides steps to configure your targets with credential brokering.
> When you use credential brokering, Boundary centrally manages credentials and returns them to the user when they attempt to connect to a target.

Reviewer: Since static cred stores can be managed by Boundary or Vault, I'd prefer this language

```suggestion
The following section provides steps to configure your targets with credential brokering.
When you use credential brokering, Boundary returns credentials to the user when they successfully connect to a target.
```

## Requirements

- You must have an existing target available.
- You must have configured either a static credential store or a Vault credential store:

  - To configure a static credential store, refer to [Create static credential stores](/boundary/docs/configuration/credential-management/static-cred-boundary).
  - To configure a Vault credential store and credential library, refer to [Create Vault credential stores](/boundary/docs/configuration/credential-management/static-cred-vault).

> - You must have a static credential saved in your static credential store or Vault credential store. The credential must correspond to the target to which you want to authenticate.

Reviewer: re-reading this, it feels a bit passive

```suggestion
- You must have a static credential saved in your static credential store or Vault credential store. The credential must correspond to the target you want to authenticate to.
```
